packages/cli/__tests__/zosjobs/__system__/download/__scripts__/download-output/download_fully_qualified.sh

```diff
@@ -22,8 +22,11 @@ then
     exit $RC
 fi
 
+# Echo the job ID for further use
+echo "Submitted job ID: $JOBID"
+
 # Loop until the job goes to the output queue
-until [ $ATTEMPTS -gt 0 ]
+until [ $ATTEMPTS -le 0 ]
```

Comment:
🤔  curious about the consideration here, and should we do this on other shell scripts?

Reply:
changing it back for consistency